velox/functions/lib/Re2Functions.cpp

```diff
@@ -196,8 +196,16 @@ class Re2MatchConstantPattern final : public VectorFunction {
     VELOX_CHECK_EQ(args.size(), 2);
     FlatVector<bool>& result = ensureWritableBool(rows, context, resultRef);
     exec::LocalDecodedVector toSearch(context, *args[0], rows);
-    checkForBadPattern(re_);
-    rows.applyToSelected([&](vector_size_t i) {
+    try {
+      checkForBadPattern(re_);
+    } catch (const std::exception& e) {
+      // Below will throw if context has noThrow set to false
```

Comment:
This comment is redundant. Let's remove. Also, there is no "noThrow" flag in context. The flag name is throwOnError.